randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -593,8 +593,13 @@ Extra - asset_id: 14
   > Door to Terminator Room
       Any of the following:
           Screw Attack or Can Lay Any Bombs
-          # 2 tap short charge from Alcatraz Escape
-          Speed Booster and Shinespark (Beginner) and Short Charge (Intermediate)
+          All of the following:
+              Speed Booster and Shinespark (Beginner)
+              Any of the following:
+                  # 2 tap short charge from Alcatraz Escape
+                  Short Charge (Intermediate)
+                  # Charge shinespark from Landing Site: https://youtu.be/AkD2kSTrV-c
+                  Movement (Intermediate)
```

Comment:
This needs no entrance rando + no door lock rando

Reply:
Right